include/mitsuba/render/endpoint.h

```diff
@@ -115,6 +115,52 @@ class MTS_EXPORT_RENDER Endpoint : public Object {
                      const Point2f &sample,
                      Mask active = true) const;
 
+    /**
+     * \brief Independently of the scene, sample a position on the endpoint.
```

Comment:
This ("independent of the scene", "independent of transport phenomena") sounds a little strange to me. Mitsuba 1 had documentation for all of these methods -- can you copy-paste from there as appropriate?